admin-tools/src/webapp/vm/user/chef_users_list.vm

```diff
@@ -76,13 +76,14 @@
 				
 
 #if ($users.size() > 0)
-			<table class="listHier lines" cellspacing="0" summary ="$tlang.getString("uselis.lisof")">
+			<table class="table table-hover table-striped table-bordered" summary ="$tlang.getString("uselis.lisof")">
 				<tr>
 					<th id="Eid">$tlang.getString("useconrem.useid")</th>
 					<th id="Name">$tlang.getString("useconrem.nam")</th>
 					<th id="Email">$tlang.getString("useconrem.ema")</th>
-					<th id="Type">$tlang.getString("usecre.typ")</th>
-					<th id="Id">$tlang.getString("useconrem.useuid")</th>
+					<th id="Type" class="hidden-xs">$tlang.getString("usecre.typ")</th>
+					<th id="Enabled" class="hidden-xs">$tlang.getString("usecre.status")</th>
```

Comment:
Minor but this id and the reference below should probably be "Status" rather than "Enabled"

Reply:
Good catch, thanks @jonespm